src/grpc/mod.rs

```diff
@@ -27,17 +27,15 @@ use base64::{
 };
 #[derive(Debug)]
 pub struct AyaCardanoRPCServer {
-    short_cache: Mutex<TimedCache<Vec<u8>,Vec<u8>>>,
-    long_cache: Mutex<TimedCache<Vec<u8>,Vec<u8>>>,
+    sized_cache: Mutex<SizedCache<Vec<u8>,Vec<u8>>>,
 }
 impl Default for AyaCardanoRPCServer {
     fn default() -> Self {
         AyaCardanoRPCServer {
-            short_cache: Mutex::new(TimedCache::with_lifespan(60)),
-            long_cache: Mutex::new(TimedCache::with_lifespan(300)),
+            sized_cache: Mutex::new(SizedCache::with_size(50)),
```

Comment:
I think it will be better if the cache size is configurable instead of hardcoded.

Reply:
Added as a config option "grpc_cache_size", and updated example_config.toml accordingly.